scripts/uploadCurrentPeriodKpis.ts

```diff
@@ -245,6 +246,51 @@ const campaigns: Campaign[] = [
       },
     ],
   },
+  {
+    protocol: 'tether-v0',
+    rewardsPeriods: [
+      {
+        startTimestamp: '2025-07-28T00:00:00Z',
+        endTimestampExclusive: '2025-08-30T00:00:00Z',
+        calculateRewards: async ({
+          resultDirectory,
+          startTimestamp,
+          endTimestampExclusive,
+        }: {
+          resultDirectory: ResultDirectory
+          startTimestamp: string
+          endTimestampExclusive: string
+        }) => {
+          await calculateRewardsTetherV0({
+            resultDirectory,
+            startTimestamp,
+            endTimestampExclusive,
+            rewardAmount: '5000000000', // 5000 USDT
+          })
+        },
+      },
+      {
+        startTimestamp: '2025-08-30T00:00:00Z',
+        endTimestampExclusive: '2025-09-30T00:00:00Z',
```

Comment:
just checking if we should make this the 1st October? it's strange that we exclude the last day of September :/